Documentation/observability/metrics.rst

```diff
@@ -301,7 +301,7 @@ Name                                       Labels
 ========================================== ===================================================================== ========== ========================================================
 ``bpf_syscall_duration_seconds``           ``operation``, ``outcome``                                            Disabled   Duration of eBPF system call performed
 ``bpf_map_ops_total``                      ``mapName`` (deprecated), ``map_name``, ``operation``, ``outcome``    Enabled    Number of eBPF map operations performed. ``mapName`` is deprecated and will be removed in 1.10. Use ``map_name`` instead.
-``bpf_map_pressure``                       ``map_name``                                                          Disabled   Map pressure defined as fill-up ratio of the map. Policy maps are exceptionally reported only when ratio is over 0.1.
+``bpf_map_pressure``                       ``map_name``                                                          Enabled    Map pressure defined as fill-up ratio of the map. Policy maps are exceptionally reported only when ratio is over 0.1.
```

Comment:
> Policy maps are exceptionally reported only when ratio is over 0.1.

Might be worth improving the wording on this while we're here, it's a bit confusing.

Reply:
Here's a suggestion:

``` 
Map pressure defined as a ratio of the map usage compared to its size. Policy map metrics are only reported when the ratio is over 0.1, ie 10% full.
```